### PR TITLE
Update playground sample with new syntax

### PR DESCRIPTION
**Pull request by arukshani**
$subject

### CODE REVIEW

File: src/hello_service.bal [hunk 1]
```diff
@@ -5,18 +5,17 @@ import ballerina/io;
 
 // A service is a network-accessible API. This service is accessed
 // at '/hello', and bound to a listener on port 9090.
-// `http:Service`is a protocol object in the `http` package.
-service<http:Service> hello bind { port: 9090 } {
+service hello on new http:Listener(9090) {
```
Comment (TylerJewell) [inline src/hello_service.bal]: Please add in the comment above explaining 'http' package and what 'http:Listener' is. This is the first example a user sees, so they will not understand that http: is package reference and it has to be explained.

File: src/hello_service.bal [hunk 2]
```diff
@@ -5,18 +5,17 @@ import ballerina/io;
...
 
-  // A resource is an invokable API method. This resource accessed
-  // at '/hello/sayHello’. `caller` is the client calling us.
-  sayHello (endpoint caller, http:Request request) {
+  //This resource accessed at '/hello/sayHello’.
```
Comment (TylerJewell) [inline src/hello_service.bal]: Dont' say "This resource" here. It assumes that the reader knows what a resource is. So instead, we have to first define a resource with something like, "A resource is an API method."

File: src/hello_service.bal (hunk 1 above)
Comment (arukshani) [reply src/hello_service.bal]: Issued new PR with the fix >> https://github.com/ballerina-guides/playground-hello-service/pull/45

File: src/hello_service.bal (hunk 2 above)
Comment (arukshani) [reply src/hello_service.bal]: Fix available in #45

**Pull request merged**